### DISCUSSION TITLE
Multiple I2C LCD displays without extra component

### DISCUSSION THREAD

**Discussion by vmynick** [arendst/Tasmota#13399] [Q&A] [answered]
Hello!

I'm looking for a solution to use multiple I2C LCD display (2x16, 4x20) on the same SDA, SCL pins without any extra component. 

I was able to display custom messages (calendar entries from Gcal by mqtt) on one display after the initialization during the start(boot).

But I not able to send any message to the 2nd display even though I changed to the correct I2C address.
After the next restart I able to communicate with the 2nd display but not with first one till the next address change and restart.

Is this what I should expect or I miss something ?

Regards, V

**Comment by arendst** [comment, marked as answer]
Not supported

**Reply by vmynick** [reply]
understood
